calicoctl/commands/version.go

```diff
@@ -44,15 +42,16 @@ func init() {
 
 func Version(args []string) error {
 	doc := `Usage:
-  <BINARY_NAME> version [--config=<CONFIG>] [--poll=<POLL>]
+  <BINARY_NAME> version [--config=<CONFIG>] [--poll=<POLL>] [--allow-version-mismatch]
 
 Options:
-  -h --help             Show this screen.
-  -c --config=<CONFIG>  Path to the file containing connection configuration in
-                        YAML or JSON format.
-                        [default: ` + constants.DefaultConfigPath + `]
-     --poll=<POLL>      Poll for changes to the cluster information at a frequency specified using POLL duration
-                        (e.g. 1s, 10m, 2h etc.). A value of 0 (the default) disables polling.
+  -h --help                 Show this screen.
+  -c --config=<CONFIG>      Path to the file containing connection configuration in
+                            YAML or JSON format.
+                            [default: ` + constants.DefaultConfigPath + `]
+     --poll=<POLL>          Poll for changes to the cluster information at a frequency specified using POLL duration
+                            (e.g. 1s, 10m, 2h etc.). A value of 0 (the default) disables polling.
+  --allow-version-mismatch  Allow client and cluster versions mismatch.
```

Comment:
nit: spacing doesn't align with `--poll` above

Reply:
Fixed, thanks for catching it!